static/app/components/events/interfaces/frame/stacktraceLink.tsx

```diff
@@ -291,7 +291,7 @@ export function StacktraceLink({frame, event, line}: StacktraceLinkProps) {
         </OpenInLink>
         {shouldshowCodecovFeatures(organization, match) && (
           <CodecovLink
-            codecovUrl={match.codecov?.coverageUrl}
+            codecovUrl={`${match.codecov?.coverageUrl}#L${frame.lineNo}`}
```

Comment:
Looks great! let's update the test for this

Reply:
Good call, thank you!